src/main/java/org/jenkinsci/test/acceptance/po/Node.java

```diff
@@ -44,6 +44,8 @@ protected Node(Injector i, URL url) {
     public abstract String getName();
 
     public void setExecutors(int n) {
+        // On high Res screens the UI redraws, so best wait for numExecutors to be present
+        waitFor(by.path("/numExecutors"));
```

Comment:
I would like to avoid doing this for every control we interact with. How comes it does not blow up on the next line?